### ISSUE TITLE
Editor showing incorrect sprite resolution

### ISSUE THREAD

**Issue by amedlock** [godotengine/godot#10779]
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
Windows 7 Professional , 64bit .  
GeForce GTX 750ti
Godot 2.1.3  64 bit  (Windows)

**Issue description:**
I have a scene (I am using as a prefab) which has an Area2D root node.  This node has a Sprite and Collision2D under it.  The sprite shows one of several sprites from a sprite sheet with 32x32 sprites in it.

In the editor the sprites look sized correctly, and the scale on the Sprite shows 3x3. In the running game however the sprites are much too big.  I have attached screen shots to show what I mean.

![editor](https://user-images.githubusercontent.com/13894868/29852898-b324a7fa-8d0a-11e7-870a-46fa2aed3e4b.png)

![in_game](https://user-images.githubusercontent.com/13894868/29852901-b8cc6e86-8d0a-11e7-8507-ed69337858fb.png)

In the top shot you can see the 9 prefabs (look like potions) and they are sized correct.  In the bottom image in game you see the image is much too large.

When I set the image on the prefab node, i set the region_rect and the modulate setting, and I don't touch the scale but its too large.  I even print the scale in game and its 3x3 so I know the editor is showing me one set of scale number but rendering another. (text output on the top image)

If I set the scale in my script to say 2x2 it looks the right scale but is off center because I centered it in the editor.

**Comment by vnen** [comment]
This is quite odd. Can you attach a minimal project that reproduces this bug?

**Comment by amedlock** [comment]
Ok, at first I couldn't reproduce in a example project so I kept looking for differences.

I see what I was doing wrong now.  I had a scaling factor on the collision2D rectangle which I was using to line up the sprite in the editor.  But of course scaling on physics objects are dropped outside of the editor.  Once I made the scale 1 and re-positioned them everything is right.

Sorry for the false alarm, and thanks for listening!
